vmdb/app/models/host.rb

```diff
@@ -789,8 +789,9 @@ def refreshable_status
   def scannable_status
     s = self.refreshable_status
     if s[:show] == false && s[:enabled] == false
-      if self.authentication_valid?(:ipmi) || !self.ipmi_address.blank?
-        if self.authentication_invalid?(:ipmi)
+      # TODO: Simplify these conditions
+      if self.has_credentials?(:ipmi) || !self.ipmi_address.blank?
+        if self.missing_credentials?(:ipmi)
```

Comment:
I think false/false should just be the same as true/false.  Basically, you didn't provide either, so at least prompt for one.